prov/efa/src/rdm/efa_rdm_pke_rta.c

```diff
@@ -301,7 +301,9 @@ int efa_rdm_pke_proc_dc_write_rta(struct efa_rdm_pke *pkt_entry)
 		EFA_WARN(FI_LOG_CQ,
 			"Posting of receipt packet failed! err=%s\n",
 			fi_strerror(err));
-		efa_rdm_rxe_handle_error(rxe, -err, FI_EFA_ERR_PKT_POST);
+		efa_base_ep_write_eq_error(&pkt_entry->ep->base_ep, err, FI_EFA_ERR_PKT_POST);
+		efa_rdm_rxe_release(rxe);
+		efa_rdm_pke_release_rx(pkt_entry);
```

Comment:
I have a bug here: `pkt_entry` should have been released by `efa_rdm_pke_proc_write_rta` already. Need to move this change after L295